doc/source/defringe_examples.rst

```diff
@@ -299,3 +299,111 @@ G750M Observation of AGK+81D266
 .. image:: output_14_0.png
 
 
+Subarray Data
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: ipython3
+    # Set up the data paths
```

Comment:
In testing this I found that there needs to be a spaces between the code:: ipython3 line and the following code in order to render the htmls correctly, the same isn't required for the parsed-literal blocks.